DEV_GUIDE.md

```diff
@@ -318,3 +319,14 @@ mvn org.asciidoctor:asciidoctor-maven-plugin:process-asciidoc@convert-to-html
 
 The output will be in `target/html/master.html`. 
 
+## Using the GitHub CI workflows against a fork
+
+All CI [workflows](.github/workflows) defined within  will execute against your fork, apart from [docker workflow](.github/workflows/docker.yml).
+To enable the docker worklflow, you need to configure two repository [variables](https://docs.github.com/en/actions/learn-github-actions/variables)
+and one respository [secret](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).
+
+* `QUAY_ORG` variable - your Quay organisation.
+* `QUAY_USER` variable - your Quay username (or robot account).
```

Comment:
We should probably default to assuming robot accounts as a hint to better security practice by default.

Reply:
docker.io doesn't have the same robot account concept as quay.io.  Instead it lets you generate access token(s) that are used in place of your password together with your regular username.

https://hub.docker.com/settings/security